influxdb/20-statefulset.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: influx
+  namespace: default
+spec:
+  podManagementPolicy: OrderedReady
+  replicas: 1
+  revisionHistoryLimit: 10
+  selector:
+    matchLabels:
+      app: influx
+  serviceName: influx
+  template:
+    metadata:
+      labels:
+        app: influx
+    spec:
+      containers:
+      - env:
+        - name: INFLUXDB_ADMIN_USER
```

Comment:
The auth is not currently working. You can log in to the DB without username and password. You might want to check the ENV variables to enable auth from https://hub.docker.com/_/influxdb